src/containers/styles/Events.scss

```diff
@@ -0,0 +1,27 @@
+$color-eng: #692D90;
+$color-eng-light: #b496c7;
+$color-eng-lightest: #e1d5e8;
+.container {
```

Comment:
Setting `.container` here will affect all containers on the site, instead set the class or id of the container in `event.jsx` or use bootstrap's spacing classes: https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
Good call, thanks.